app/scripts/templates/settings.mustache

```diff
@@ -1,24 +1,30 @@
 <div id="fxa-settings-header-wrapper">
   <header id="fxa-settings-header">
-    <h1 id="fxa-manage-account">{{#t}}Manage account{{/t}}</h1>
+    <h1 id="fxa-manage-account">{{#t}}Firefox Account{{/t}}</h1>
     {{#showSignOut}}
     <button id="signout" class="settings-button secondary">{{#t}}Sign out{{/t}}</button>
     {{/showSignOut}}
   </header>
+
   <div class="settings-success-wrapper">
     <div class="success settings-success">Hello</div>
```

Comment:
Do we still need this `Hello` here?
1. it seems like it is not visible
2. the contents of that get replaced with some "updated" message. 

So should this just be ` <div class="success settings-success"></div>` ?